spec/perimeter_x_cookie_validator_spec.rb

```diff
@@ -72,7 +72,7 @@
     verified, px_ctx = validator.verify(px_ctx)
     expect(verified).to eq false
     expect(px_ctx.context[:s2s_call_reason]).to eq PxModule::COOKIE_DECRYPTION_FAILED
-    expect(px_ctx.context[:px_orig_cookie]).to eq @req.cookies[:_px]
```

Comment:
This is the potentially breaking change to which I was referring. Because decryption failed, we cannot necessarily assume that the cookie is in a certain format.

If `px_orig_cookie` needs to remain a string, we can change the setting code to something like:

```ruby
px_ctx.context[:px_orig_cookie] = cookie ? cookie.px_cookie : px_ctx.context[:px_cookie]
```